src/App.tsx

```diff
@@ -1,39 +1,59 @@
-import React from 'react';
+import { Component } from 'react';
```

Comment:
Consider importing React as well since you're using JSX in your render method. For example: import React, { Component } from 'react';